lib/PrimitiveHelpers.js

```diff
@@ -63,6 +78,14 @@ function primitivesHaveOverlappingIndexAccessors(gltf, primitive, comparePrimiti
     return false;
 }
 
+/**
```

Comment:
This is copy pasted from `primitivesHaveOverlappingIndexAccessors` but still has the same header and return comment not matching this function.